### PR TITLE
Dedicated pools for KafkaActorPublisher and KafkaActorSubscriber

### PR DESCRIPTION
**Pull request by amorfis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,10 @@ val subscriber = kafka.publish("uppercaseStrings", "groupName")
 
 Source(publisher).map(_.toUpperCase).to(Sink(subscriber)).run()
 ```
+
+Tuning
+----
+
+KafkaActorSubscriber and KafkaActorPublisher have their own thread pools, configured in `application.conf`.
```
Comment (ktoso) [inline README.md]: Hey there @amorfis, as a library this repo should *not* contain any `application.conf` files.
Instead it should provide `reference.conf` with "default values" (exactly the contents of your current application.conf).
The reason for this is that users of your library will provide `application.conf` files which *must* take precedence over reference conf - so they are able to override your defaults. Refer to https://github.com/typesafehub/config#longer-examples for more rationale

File: src/main/scala/com/softwaremill/react/kafka/ReactiveKafka.scala [hunk 2]
```diff
@@ -23,7 +23,7 @@ class ReactiveKafka(val host: String, val zooKeeperHost: String) {
 
   def consumeAsActor(topic: String, groupId: String)(implicit actorSystem: ActorSystem): ActorRef = {
     val consumer = new KafkaConsumer(topic, groupId, zooKeeperHost)
-    actorSystem.actorOf(Props(new KafkaActorPublisher(consumer)))
+    actorSystem.actorOf(Props(new KafkaActorPublisher(consumer)).withDispatcher("kafka-publisher-dispatcher"))
```
Comment (ktoso) [inline src/main/scala/com/softwaremill/react/kafka/ReactiveKafka.scala]: yup!

File: src/main/resources/application.conf [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    # No of core threads ... ceil(available processors * factor)
+    core-pool-size-factor = 2.0
+    # maximum number of threads to cap factor-based number to
+    core-pool-size-max = 20
+  }
+  # Throughput defines the maximum number of messages to be
+  # processed per actor before the thread jumps to the next actor.
+  # Set to 1 for as fair as possible.
+  throughput = 1
+}
```
Comment (ktoso) [inline src/main/resources/application.conf]: sounds good for a start, some tuning may be a good idea to find nice defaults here, but that can be up to the user for now.

**Comment by ktoso** [comment]
application.conf should be renamed to reference.conf, other than that that's already a great move forward :)

File: README.md (hunk 1 above)
Comment (amorfis) [inline README.md]: Thank you Konrad! Change pushed :)

**Comment by kciesielski** [comment]
Please correct also the `conf` file name in `README.md`.

**Comment by amorfis** [comment]
Done

**Pull request merged**